### DISCUSSION TITLE
Draft roles lists

### DISCUSSION THREAD

**Discussion by monotasker** [MESH-Research/knowledge-commons-repository#86] [Ideas]
Here are draft lists of roles for creators and contributors. There's significant overlap, since in some cases a role (e.g., translator) might be the primary producer of a deposit and in other cases that same role might be considered secondary.

CREATOR ROLES
- analyst
- arranger
- artisan
- artist
- author
- author of introduction
- author of forward
- author of afterward
- designer
- illustrator
- choreographer
- cinematographer
- composer
- conductor
- director
- editor
- interviewee
- interviewer
- inventor
- lyricist
- manufacturer
- performer
- photographer
- producer
- programmer
- project or team leader
- project or team member
- reporter
- researcher
- screenplay
- speaker
- translator
- research group
- translator
- other

**Comment by monotasker** [comment]
CONTRIBUTOR ROLES

- actor
- adaptor
- advisor
- analyst
- annotator
- arranger
- assistant
- attributed name
- artisan
- author of introduction
- author of foreword
- author of afterward
- choreographer
- cinematographer
- collaborator
- collector
- committee chair
- committee member
- composer
- conductor
- consultant
- contact person
- correspondent
- data collector
- data curator
- data manager
- dedicatee
- designer
- distributor
- donor
- drafter
- editor
- examiner
- former owner
- hosting institution
- illustrator
- interviewee
- interviewer
- juror
- licensee
- licensor
- lyricist
- manufacturer
- organizer
- owner
- performer
- photographer
- printer
- producer
- project manager
- proofreader
- recording engineer
- referee
- reporter
- research assistant
- rights holder
- project or team leader
- project or team member
- registration agency
- registration authority
- related person
- researcher
- rights holder
- screenplay
- speaker
- sponsor
- supervisor
- transcriber
- translator
- witness
- writer of accompanying material
- video editor
- work package leader
- other

**Comment by bjr70** [comment]
@stephanievaskomsu @babaklar Are there too many? Is it overwhelming? I don't know. I like the idea of specificity, but at the same time I'm not sure. 

Example: 

- author of introduction
- author of foreword
- author of afterward

I would probably not have those in creator and instead have them in contributor. They are not the main author of the work, if that makes sense.

**Reply by monotasker** [reply]
Yeah, I wasn't sure about those either. You're probably right that they don't make sense in "creator".

On the issue of it being overwhelming: My thought is to (a) group related roles in the selector, and (b) restrict the roles we display based on resource type. So "choreographer" would show up on "performance," but not on "book."

**Reply by babaklar** [reply]
Not gonna lie, I feel like the list is a bit overwhelming--I can see users getting frustrated by how many options they have to scroll through. However, I also think if they're restricted based on resource type like Ian suggested that would definitely help. It's hard to know right now whether users would appreciate all these options or if they'll also find them overwhelming.

**Reply by bjr70** [reply]
Yeah, if we can cut the list down based on the item type that would do a lot to make this less unwieldy.

**Reply by stephanievaskomsu** [reply]
I also got overwhelmed, but I'm also overwhelmed about Ian's idea even though I think it's good in theory. It may just be a case of unknown unknowns, and when we miss a role for something that should have been there, the depositor can let us know? Speaking of that---big block at the bottom of all pages that says "Questions? Comments? Contact us at hello@commons.org" maybe?

**Reply by monotasker** [reply]
Definitely +1 to the footer with contact info

**Reply by monotasker** [reply]
Maybe what I can do is go ahead with a long list like this to start, restricting it by resource type. Then we can see how usable it is. We can always chop it after some testing.
